components/resume/themes/DEAFULT_THEME/sections/ExperienceSection.tsx

```
@@ -25,30 +25,30 @@ const ExperienceSection = ({section, previousSectionType}: ExperienceSectionProp
 			<p className="text-sm italic">
-				{experience.jobTitle && <span>{experience.jobTitle}</span>}
-				{experience.jobTitle && experience.employmentType && <span>, </span>}
-				{experience.jobTitle && <span>{experience.employmentType}</span>}
+				{experience.job_title && <span>{experience.job_title}</span>}
+				{experience.job_title && experience.employment_type && <span>, </span>}
+				{experience.job_title && <span>{experience.employment_type}</span>}
 			</p>
 
 			{/* DESCRIPTION */}
-			{experience.description && <p className="text-sm leading-snug mt-0.5 pl-8">{experience.description}</p>}
+			{experience.description && <div className="text-sm mt-0.5 pl-2 prose prose-sm leading-snug prose-p:m-0 max-w-none" dangerouslySetInnerHTML={{__html: experience.description}}/>}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)